layers/+email/notmuch/funcs.el

```diff
@@ -105,7 +105,7 @@ messages in the current thread"
          (browse-url url)))
       (diff-mode)
       (view-mode 1)
-      (pop-to-buffer (current-buffer)))))
+      (switch-to-buffer (current-buffer)))))
```

Comment:
Would using the `pop-to-buffer-same-window` function achieve the desired effect?

Reply:
I think this would be better..